schema/field/field.go

```diff
@@ -549,6 +549,13 @@ func (b *bytesBuilder) Optional() *bytesBuilder {
 	return b
 }
 
+// Unique makes the field unique within all vertices of this type.
+// Only supported in PostgresSQL.
```

Comment:
```suggestion
// Only supported in PostgreSQL.
```

Found a typo after merging it if you want to send a patch @tarrencev